src/wodan/wodan.ml

```diff
@@ -199,23 +199,31 @@ end
 
 type logdata_index = {
   contents : string KeyedMap.t;
-  (* The last value_end that was read from or written to disk *)
```

Comment:
Can we keep this comment ?

Reply:
It applied to the line which was removed.
I expect it was in the wrong place due to an ocamlformat issue.

Reply:
Oh, right :)